tests/b018_classes.py

```diff
@@ -30,3 +30,4 @@ class Foo3:
     123
     a = 2
     "str"
+    1
```

Comment:
Did we error at all without these ints? Why did you add them? Just trying to understand.

Reply:
As you can see in the tests line 32 is ignored and to test if something is found, I added line 33

Reply:
So this is already a working test

Reply:
Yeah, this tests we're ignoring strs as a whole, I was more stating while we have examples that are causing the noise lets add those into this file too. But it can be next PR as well.

Reply:
Ah, you mean the docstring. Valid point. I'll take a look what we can improve